.cirrus.yml

```diff
@@ -134,8 +135,9 @@ task:
   ##   - rm /tmp/.com.apple.dt.CommandLineTools.installondemand.in-progress
   ##
   brew_valgrind_pre_script:
+    - brew update
     - brew config
-    - brew tap --shallow LouisBrunner/valgrind
+    - brew tap LouisBrunner/valgrind
```

Comment:
Why not shallow?

Reply:
Shallow doesn't exist anymore in recent brew versions, see the commit message.